kratos/python_scripts/deprecation_management.py

```diff
@@ -0,0 +1,37 @@
+import KratosMultiphysics as KM
+
+class DeprecationManager:
+
+    @classmethod
+    def HasDeprecatedVariable(cls, context_string, settings, old_variable_name, new_variable_name):
```

Comment:
```suggestion
    @staticmethod
    def HasDeprecatedVariable(context_string, settings, old_variable_name, new_variable_name):
```
this seems more appropriate here

Reply:
Ok, initially I was using the cls to pull out the string and I left it there...